alot/commands/globals.py

```diff
@@ -733,7 +733,8 @@ def apply(self, ui):
                           priority='error')
                 return
             try:
-                self.envelope.parse_template(open(path).read())
+                with open(path) as f:
+                    self.envelope.parse_template(f.read())
```

Comment:
Thanks, in the meanwhile I also read both commit messages.

Now I am all for it!